sentinelone_connector.py

```diff
@@ -417,29 +687,587 @@ def _handle_get_threat_info(self, param):
         header["Authorization"] = "APIToken %s" % self.token
         params = {"ids": [s1_threat_id]}
         ret_val, response = self._make_rest_call('/web/api/v2.1/threats', action_result, headers=header, params=params)
+        action_result.add_data(response)
+        self.save_progress("Ret_val: {0}".format(ret_val))
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_applications(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        ip_hostname = param.get('ip_hostname')
+        params = None
+        if ip_hostname:
+            try:
+                ret_val = self._get_computer_name(ip_hostname, action_result)
+            except Exception:
+                return action_result.set_status(phantom.APP_ERROR, "Did not get proper response from the server")
+            self.save_progress('Agent query: {}'.format(ret_val))
+
```

Comment:
-1 handling is required or not? Pls check and do the needfull.

Reply:
Done